backend/utils/nunjucksSetup.ts

```diff
@@ -47,6 +47,13 @@ export default (app) => {
 
   njkEnv.addFilter('formatDate', (value, format) => (value ? moment(value).format(format) : null))
 
+  njkEnv.addFilter('checkValidAndFormatDate', (value, format) => {
```

Comment:
Not sure about the name - maybe `getSortableDate` would make more sense?

Actually looking at where it is used, it is only called if there isn't a date - so maybe `defaultSortDate`?

Reply:
Will rename this, defaultSortDate works - it just wouldn't sort when there was text in some cells and dates in others